src/axom/sidre/examples/sidre_createdatastore.cpp

```diff
@@ -415,6 +415,60 @@ void generate_blueprint(DataStore* ds) {
   // _blueprint_generate_save_end
 }
 
+void generate_blueprint_to_path(DataStore* ds) {
+  // _blueprint_generate_path_start
+  // Conduit needs a specific hierarchy.
+  // For this example, we locate the domain data at a path
+  // that is not the top level of its Sidre heirarchy.
+  DataStore cds;
+  std::string domain_name = "domain";
+  std::string domain_location = "domain_data/level/domains/" + domain_name;
+  std::string mesh_name = "mesh";
+  std::string domain_mesh = domain_location + "/" + mesh_name;
+
+  Group* mroot = cds.getRoot()->createGroup(domain_location);
+  Group* coords = mroot->createGroup(mesh_name + "/coordsets/coords");
+  Group* topos = mroot->createGroup(mesh_name + "/topologies");
+  // no material sets in this example
+  Group* fields = mroot->createGroup(mesh_name + "/fields");
+  // no adjacency sets in this (single-domain) example
+  // _blueprint_generate_path_end
+
+  setup_blueprint_coords(ds, coords);
+
+  setup_blueprint_topos(ds, topos);
+
+  setup_blueprint_fields(ds, fields);
+
+  // _blueprint_generate_path_save_start
+  conduit::Node info, mesh_node, root_node;
+  cds.getRoot()->createNativeLayout(mesh_node);
+  if (conduit::blueprint::verify("mesh", mesh_node[domain_mesh], info))
+  {
+    std::string bp("rootfile_data/blueprint_index/automesh");
```

Comment:
How does the name "automesh" relate the the mesh in the hierarchy named "mesh"?

Will this show up in visit as "automesh", even though it is named "mesh" in the actual hierarchy?

Reply:
This will show up as "automesh" in VisIt.  As I understand it, there is no requirement that the name held by the blueprint index has to match the name of the mesh in the hierarchy.  Some of the other examples in this file do have the names matching.